docs/source/guides/export_model.mdx

```diff
@@ -421,7 +421,7 @@ The NeuronX TGI image includes not only NeuronX runtime, but also all packages a
 Use the following command to export a model to Neuron using a TGI image:
 
 ```
-docker run --emtrypoint optimum-cli \
+docker run --entrypoint optimum-cli \
```

Comment:
Thank you for that ! I noticed it also when I added the link in the Llama 70B issue.